release-notes/PTP/PTP_1.1.3_CU2.md

```diff
@@ -0,0 +1,78 @@
+---
+uid: PTP_1.1.3_CU2
+---
+
+# PTP 1.1.3 CU2
+
+## Enhancements
+
+#### New parameters added to standard PTP mediation layer [ID_34993]
```

Comment:
Could you rewrite this in the following way please?

#### Improved support for Arista Manager [ID_34993]

The following parameters from the Arista Manager connector have been added to the mediation layer:

- Mode, which represent the switch clock mode, is now mapped to PID 33001 of the Arista Manager.

Reply:
Done!